internal/pkg/agent/application/application.go

```diff
@@ -111,10 +116,18 @@ func New(
 	}
 	monitor := monitoring.New(isMonitoringSupported, cfg.Settings.DownloadConfig.OS(), cfg.Settings.MonitoringConfig, agentInfo)
 
+	controlAddress := control.Address()
+	log.Debugf("Control address: %s", controlAddress)
+	address, err := deriveCommsAddress(controlAddress, cfg.Settings.GRPC)
+	if err != nil {
+		return nil, nil, nil, fmt.Errorf("failed to derive comms GRPC: %w", err)
+	}
+	log.Debugf("Comms GRPS address: %s", address)
```

Comment:
I would prefer this to be a nice looking message that is at the info level. 

https://github.com/elastic/elastic-agent/blob/main/pkg/control/v2/server/server.go#L88